SAP/spartacus
# fix: cannot recognize current semantic route for my company pages

Previously, the routing config chunks for my company pages were provided in the components modules, which were lazy loaded. This caused 2 issues:
- We could not recognize the semantic route for any my-accunt page (because `SemanticRoutesService.getRouteName()` depends on the routing config provided in the root injector on the app start, but not on configs provided lazily).
-  it was impossible to create a link to any my-company page (using `cxUrl` pipe) from another pages outside the my-company feature. (Currently we don't need such a links in OOTB Spartacus, but it's a potential future limitation.)

Now above 2 issues are fixed, as we moved all the default routing config chunks to the `OrganizationAdministrationRootModule`, which provides the config chunks in the root injector. Therefore they are available in the `SemanticRoutesService` on app start.

### QA steps
1. Add in the constructor of the `StorefrontComponent` this snippet (log the semantic route on page change):
    ```ts
    this.routingService
      .getRouterState()
      .pipe(
        map((state) => state.state.semanticRoute),
        distinctUntilChanged(),
        tap((semanticRoute) => console.log({ semanticRoute }))
      )
      .subscribe();
    ```
2. Turn on 'b2b' in `environment.ts`
3. Login as `linda.wolf@rustic-hw.com`
4. Navigate through my-company pages and verify the correct semantic route name is logged in the console for each feature's pages: budgets, cost centers, user groups, users, permissions, units

Note: Some content pages like FAQ, find a store or help don't have route config provided - therefore it's a valid result that they `undefined` semantic route is logged in the console for those pages. 

----
fixes #12717
